python/pyspark/sql/connect/dataframe.py

```diff
@@ -305,8 +308,12 @@ def _print_plan(self) -> str:
             return self._plan.print()
         return ""
 
-    def collect(self) -> None:
-        raise NotImplementedError("Please use toPandas().")
+    def collect(self) -> List[Row]:
+        pdf = self.toPandas()
+        if pdf is not None:
+            return list(pdf.apply(lambda fun: Row(*fun), axis=1))
```

Comment:
```suggestion
            return list(pdf.apply(lambda row: Row(*row), axis=1))
```

Reply:
Should probably need to set its names from the schema. e.g., 

```python
>>> Row(a=1).a
1
```

should work.

Reply:
Done. I found magic is to use `**` so that each lambda row will become `kwargs`. 

Honestly I don't understand this magic but it is working at least.